sdk/ai/ai-inference-rest/package.json

```diff
@@ -114,7 +114,7 @@
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
     "build:samples": "echo Obsolete",
-    "generate": "tsp-client update",
+    "generate": "rimraf ./src/generated && mkdirp ./src/generated && cp tsp-location.yaml ./src/generated && tsp-client update -o ./src/generated && rm ./src/generated/tsp-location.yaml",
```

Comment:
would we want to use rimraf for the last part too instead of using rm?

Reply:
[Updated](https://github.com/Azure/azure-sdk-for-js/pull/30785/commits/c31401930613857354b5d84e785e026c3414434e)